.github/workflows/check-stage.yaml

```
@@ -0,0 +1,27 @@
+name: Validate Commit Was Deployed to Staging
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)